### DISCUSSION TITLE
Case insensitive numeric hint filter

### DISCUSSION THREAD

**Discussion by chriswu000** [brookhong/Surfingkeys#1462] [Ideas]
Hi!

I love surfingkeys!  I've recently switched to using numeric hints since that's what I am used to from previous vim browser plugins, and also because it lets me start filtering as soon as I hit "f".  It makes it feel like I'm having a conversation with the browser rather than needing to review what the letter hints are each time I hit "f".

The only problem is that I'd prefer that the filtering were case insensitive.  Is there any way to easily add that as a setting?  If it's not simple, I can take a look at submitting a pull request, but I have not really worked much with Chrome plugins, so that might take quite a long time.  Thanks!

Best,
Chris

**Comment by userrand** [comment]
> Hi!
> 
> I love surfingkeys! I've recently switched to using numeric hints since that's what I am used to from previous vim browser plugins, and also because it lets me start filtering as soon as I hit "f". It makes it feel like I'm having a conversation with the browser rather than needing to review what the letter hints are each time I hit "f".
> 
> The only problem is that I'd prefer that the filtering were case insensitive. Is there any way to easily add that as a setting? If it's not simple, I can take a look at submitting a pull request, but I have not really worked much with Chrome plugins, so that might take quite a long time. Thanks!
> 
> Best, Chris

Hi, I am scrolling through discussions and saw this one. Regex seems to be supported with `/` so you can search for [fF] to search for f or F.
